tests/behat/behat-features/replication-aditions.feature

```diff
@@ -0,0 +1,208 @@
+@api
+@replication @replication-aditions
+Feature: Contentpool media replication basically works.
+
+  Make sure that media referenced in content can be pulled from remote contentpool server.
+
+  Scenario:
```

Comment:
This needs to be marked as background scenario still to ensure it's run:
http://behat.org/en/latest/user_guide/writing_scenarios.html#backgrounds

Reply:
When marked as background i get:
```Background:                                                                                                                                           # behat-features/replication-aditions.feature:7
    Given I am logged in to contentpool                                                                                                                 # ContentPoolContext::loggedInToContentpool()
    And I visit path "media/add/image" on contentpool                                                                                                   # ContentPoolContext::visitContentpoolPath()
    And I fill in "name[0][value]" with "BEHAT: Media image 1"                                                                                          # Drupal\DrupalExtension\Context\MinkContext::fillField()
    When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]" # Drupal\DrupalExtension\Context\MinkContext::attachFileToField()
    And I press "Save and publish"                                                                                                                      # Drupal\DrupalExtension\Context\MinkContext::pressButton()
      Browser crashed (Behat\Mink\Exception\DriverException)
    │
    │  Current URL: 
    │
    ╳  Chrome is unreachable via "ws://chrome:9222/devtools/page/27D84381B5850460522D8080335BDC8F" and might have crashed. Please see docs/troubleshooting.md (RuntimeException)
    │
    └─ @AfterStep```

Reply:
if not - nevermind

Reply:
With @javascript on file level:
```
When I attach the file "core/profiles/demo_umami/modules/demo_umami_content/default_content/images/chili-sauce-umami.jpg" to "files[field_image_0]" # Drupal\DrupalExtension\Context\MinkContext::attachFileToField()
    ┌─ @BeforeStep # Drupal\DrupalExtension\Context\MinkContext::beforeJavascriptStep()
    │
    ╳  Browser crashed (Behat\Mink\Exception\DriverException)
```

File was only attached when it was a scenario without javascript. And file is required to create media so can't skip this step.